MovingBlocks/Terasology
# Fix checkstyle violations

### Contains

Fixes all checkstyle error violations.

### How to test

Run ./gradlew checkstyleMain or open the project in eclipse with the checkstyle plugin active (should also work with any other IDE).

### Critical parts

- I added an implementation for equals() in ChunkImpl because the hashCode() method is overwritten
- I renamed some constants in /engine/src/main/java/org/terasology/rendering/openvrprovider/ControllerListener.java
- I changed a lambda expression to an anonymous class instantiation as the checkstyle tool seems to get confused by variable declarations inside lambdas (in my defense, this was a big lambda function so in my opinion it's not really worse to use an anonymous class) -> /engine/src/main/java/org/terasology/logic/console/commandSystem/ConsoleCommand.java
